TemplePlus/python/python_object.cpp

```diff
@@ -2443,6 +2444,11 @@ static PyObject* PyObjHandle_Unwield(PyObject* obj, PyObject* args) {
 		es = EquipSlot::Invalid;
 	auto equipSlot = (EquipSlot)es;
 
+	if (item != objHndl::null && inventory.GetParent(item) == self->handle ) {
```

Comment:
Yes, simply the usage of the optional argument is what triggers it. I figure if you don't want to drop an item, you don't put an item there, and if you do, the item handle is the minimum information required to call drop. Should I instead check that the item is first in an equip slot instead of just the inventory?